apps/web/components/AppListCard.tsx

```diff
@@ -65,7 +66,13 @@ export default function AppListCard(props: AppListCardProps) {
   return (
     <div className={`${highlight ? "dark:bg-muted bg-yellow-100" : ""}`}>
       <div className="flex gap-x-3 px-5 py-4">
-        {logo ? <img className="h-10 w-10" src={logo} alt={`${title} logo`} /> : null}
+        {logo ? (
+          <img
```

Comment:
While I was testing, I discovered the same problem is here as well.
<img width="1281" alt="Screenshot 2023-05-19 at 11 31 02 AM" src="https://github.com/calcom/cal.com/assets/1780212/600808e9-1205-44fc-8845-3e19eb60610c">
<img width="696" alt="Screenshot 2023-05-19 at 11 35 20 AM" src="https://github.com/calcom/cal.com/assets/1780212/a7a82137-c9bc-40f3-9161-641f6788836e">